### PR TITLE
Added items prop to Cv-Dropdown

### LINKED ISSUE #951: How to populate cv-dropdown component options?
Hi there,
Thanks for the great components. I'm currently migrating from Vuetify and wondering if there is a better solution to populating the cv-dropdown-item? It doesn't seem to be in the docs.

What I am doing now.
`<cv-dropdown-item v-bind:key="option.Id" v-for="option in reportSources" :value="option.Source">{{option.Title}}</cv-dropdown-item>
              </cv-dropdown>`

Thanks!

### PR DESCRIPTION
**Pull request by DakotaWray2**
Closes #951

Added items prop to Cv-Dropdown

#### Changelog

A       .yarn-offline-mirror/bn.js-4.11.9.tgz
A       .yarn-offline-mirror/elliptic-6.5.3.tgz
M       package.json
M       packages/core/src/components/cv-dropdown/cv-dropdown.vue
M       storybook/stories/cv-dropdown-story.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "prepare": "lerna bootstrap",
     "prepublishOnly": "yarn build",
     "reinstall": "echo Clearing yarn cache; yarn cache clean; echo Removing yarn offline mirror; rimraf .yarn-offline-mirror; echo Removing package folders; lerna clean --yes; rimraf node_modules; yarn install",
-    "start": "yarn info @carbon/vue --json > package-info.jsonl; vue-cli-service storybook:serve -p 9001 -c storybook/.storybook",
+    "start:a": "yarn info @carbon/vue --json > package-info.jsonl;",
```
Comment (lee-chase) [inline package.json:21]: Why have the start lines been split?

File: packages/core/src/components/cv-dropdown/cv-dropdown.vue [hunk 2]
```diff
@@ -88,7 +88,10 @@
           :aria-labelledby="`${uid}-label`"
           ref="droplist"
         >
-          <slot></slot>
+          <slot v-if="!this.$props.items"></slot>
```
Comment (lee-chase) [inline packages/core/src/components/cv-dropdown/cv-dropdown.vue:91]: Typically we have used the form where the slot overrides the prop content. 
```
<slot name="items">{{items}}</slot>
```

Do not access this.$props.items, use items directly e.g. `v-for="item in items"`

File: package.json (hunk 1 above)
Comment (DakotaWray2) [reply package.json:21]: I had an issue running the original start command on Windows. found the solution in your comment on #910

File: packages/core/src/components/cv-dropdown/cv-dropdown.vue (hunk 2 above)
Comment (DakotaWray2) [reply packages/core/src/components/cv-dropdown/cv-dropdown.vue]: Will resubmit pull request with change

**Pull request closed without merging**